dotnet/samples/01.messaging.echoBot/README.md

```diff
@@ -0,0 +1,38 @@
+﻿# Teams Conversation Bot
+
+Teams AI  Conversation Bot sample for Teams.
+
+This sample shows how to incorporate basic conversational flow into a Teams application. 
+It also illustrates a few of the Teams specific calls you can make from your bot.
+
+## Set up instructions
+
+All the samples in for the C# .NET SDK can be set up in the same way: You can find the step by step instructions here:
+ [Setup Instructions](../../README.md).
+
+## Interacting with the Bot
+
+At this point you should have set up the bot and installed it in Teams. You can interact with the bot by sending it a message.
+
+Here's a sample interaction with the bot:
+
+![Sample interaction](assets/helloworld.png)
+
+You can reset the message count by sending the bot the message `/reset`.
+
+![Reset interaction](assets/reset.png)
+
+## Deploy to Azure with Teams Toolkit for Visual Studio
```

Comment:
"Deploy to Azure" is fine because the following contents include the CLI way for deployment as well.

Reply:
resolved